resources/[soz]/soz-core/src/shared/shop/superette.ts

```diff
@@ -0,0 +1,73 @@
+import { ShopBrand } from '@public/config/shops';
+
+import { InventoryItemMetadata, Item, ItemType } from '../item';
+import { PlayerLicenceType } from '../player';
+import { ShopProduct } from '../shop';
+
+export type ShopContent = ShopProduct[];
+
+export type SuperetteItem = Item & {
+    price?: number;
+    amount?: number;
+    slot?: number;
+};
+
+export type ShopItem = {
+    name: string;
+    label: string;
+    description: string;
+    weight: number;
+    slot: number;
+    useable: boolean;
+    unique: boolean;
+    type: ItemType;
+    amount: number;
+    metadata?: InventoryItemMetadata;
+    shouldClose?: boolean;
+    illustrator?: Record<string, string> | string;
+    disabled?: boolean; // added by inventory on the fly
+    shortcut?: string | null; // added by inventory on the fly
+    price: number;
+};
```

Comment:
Prefer use this 
```ts
export type ShopItem = InventoryItem & {
    price: number;
}
```

Reply:
This type is not used anymore, replaced by `CartItem` declared in `resources\[soz]\soz-core\src\shared\shop\superette.ts`
I deleted this type

Reply:
Done